vxfreeswitch/voice.py

```diff
@@ -370,8 +370,11 @@ def send_outbound_message(self, client, message):
 
         if overrideURL is None:
             yield client.output_message("%s\n" % content)
-        else:
+        elif isinstance(overrideURL, basestring):
             yield client.output_stream(overrideURL)
+        elif isinstance(overrideURL, list):
+            for url in overrideURL:
+                yield client.output_stream(url)
```

Comment:
Should we add an `else` clause that logs an error if `overrideURL` is something unexpected?